environment.yml

```diff
@@ -0,0 +1,12 @@
+name: movingpandas
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.7
+  - numpy
+  - matplotlib
+  - seaborn
+  - pandas
+  - geopandas
+  - contextily
```

Comment:
You can add more packages here. I usually don't pin exactly, like you had in the `requirements.txt` unless it is really needed.